wplib.php

```diff
@@ -375,6 +508,20 @@ private static function _find_autoload_files() {
 	}
 
 	/**
+	 * Force loading of all classes if needed to find all classes with a specific constant.
+	 */
+	static function autoload_all_classes() {
```

Comment:
It's only run to prime cached data.  It's needed to file all `WPLib_Post_Base::POST_TYPE` defined in code, and also to verify that all files to be auto-loaded are recognized and added to the cache.

There may be other ways we can enhance it only run in development mode but for now -- assuming a persistent cache -- it should only run for one page-load each time the cache is cleared.